### ISSUE TITLE
hypothesis.errors.HypothesisException: The $HYPOTHESIS_DATABASE_FILE environment variable no longer has any effect.

### ISSUE THREAD

**Issue by legrostdg** [pimutils/todoman#379]
Hi,

`todoman` fails to build on Debian:  https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=952161

http://qa-logs.debian.net/2020/02/22/todoman_3.5.0-2_unstable.log

```
...
The manual pages are in build/sphinx/man.
   dh_auto_test -O--buildsystem=pybuild
I: pybuild base:217: cd /<<PKGBUILDDIR>>/.pybuild/cpython3_3.8/build; python3.8 -m pytest tests
/<<PKGBUILDDIR>>/.pybuild/cpython3_3.8/build/todoman/formatters.py:133: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if priority is None or priority is '':
ImportError while loading conftest '/<<PKGBUILDDIR>>/.pybuild/cpython3_3.8/build/tests/conftest.py'.
tests/conftest.py:165: in <module>
    settings(
/usr/lib/python3/dist-packages/hypothesis/_settings.py:175: in __init__
    kwargs[setting.name] = getattr(defaults, setting.name)
/usr/lib/python3/dist-packages/hypothesis/_settings.py:75: in __get__
    result = ExampleDatabase(not_set)
/usr/lib/python3/dist-packages/hypothesis/database.py:61: in __call__
    return _db_for_path(*args, **kwargs)
/usr/lib/python3/dist-packages/hypothesis/database.py:34: in _db_for_path
    raise HypothesisException(
E   hypothesis.errors.HypothesisException: The $HYPOTHESIS_DATABASE_FILE environment variable no longer has any effect.  Configure your database location via a settings profile instead.
E   https://hypothesis.readthedocs.io/en/latest/settings.html#settings-profiles
E: pybuild pybuild:341: test: plugin distutils failed with: exit code=4: cd /<<PKGBUILDDIR>>/.pybuild/cpython3_3.8/build; python3.8 -m pytest tests
dh_auto_test: error: pybuild --test --test-pytest -i python{version} -p "3.8 3.7" returned exit code 13
make: *** [debian/rules:28: build] Error 25
dpkg-buildpackage: error: debian/rules build subprocess returned exit status 2

**Comment by WhyNotHugo** [comment]
Can you provide the output of `pip install`?

**Comment by legrostdg** [comment]
The package is built only from other Debian packages. Maybe you want the version of hypothesis available on Debian? python3-hypothesis 4.36.2-1

**Comment by legrostdg** [comment]
I investigated a bit further and found the problem, which was in the packaging: we did set HYPOTHESIS_DATABASE_FILE to prevent the installation of the Hypothesis example file, to make the build reproducible. It does not seem to be useful anymore. Sorry about the false alert!
